Alignment/HIPAlignmentAlgorithm/python/cdcsTrackSelection_cff_py.txt

```diff
@@ -0,0 +1,25 @@
+import Alignment.CommonAlignment.tools.trackselectionRefitting as TrackRefitterSequencer
```

Comment:
These are template files that are later replaced inside the actual align_cfg.py of each subjob. This is why their extension was made .txt (even before I was in CMS).